services/github/github-followers.service.js

```diff
@@ -25,7 +25,15 @@ module.exports = class GithubFollowers extends LegacyService {
     return [
       {
         title: 'GitHub followers',
-        previewUrl: 'espadrine',
+        pattern: ':user',
+        namedParams: {
+          user: 'espadrine',
+        },
+        staticPreview: {
+          label: 'Follow',
+          message: '150',
+          // style: 'social',
```

Comment:
^^ This probably needs to be fixed before this is merged. Or else this one could be reverted.